.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### Description
+
+[Description of the bug or feature]
+
+### Steps to reproduce
+
+1. [First step]
+2. [Second step]
+3. [and so on...]
+
+**Expected behavior:** [What you expected to happen]
+
+**Actual behavior:** [What actually happened]
+
+**Link to [JSFiddle](https://jsfiddle.net/):** [If you're reporting a bug that's not reproducible on our [demo page](https://yabwe.github.io/medium-editor/demo.html), please try to reproduce it on [JSFiddle](https://jsfiddle.net/) and paste a link here]
```

Comment:
Might be worth calling out some of the other options people use as well (ie jsbin, codepen, etc.)